e2e/cypress/integration/accessibility/accessibility_keyboard_usability_spec.js

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @accessibility
+
+import users from '../../fixtures/users.json';
+
+const otherUser = users['user-2'];
+const count = 5;
+let message;
+
+function postMessages(cnt = 1) {
+    cy.getCurrentChannelId().then((channelId) => {
+        cy.apiGetUserByEmail(otherUser.email).then((emailResponse) => {
+            cy.apiAddUserToChannel(channelId, emailResponse.body.id);
+            for (let index = 0; index < cnt; index++) {
+                // # Post Message as Current user
+                message = `hello from sysadmin: ${Date.now()}`;
+                cy.postMessage(message);
+                message = `hello from ${otherUser.username}: ${Date.now()}`;
+                cy.postMessageAs({sender: otherUser, message, channelId});
+            }
+            cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
+        });
+    });
+}
+
+describe('Verify Accessibility keyboad usability across different regions in the app', () => {
+    before(() => {
+        cy.apiLogin('sysadmin');
+
+        // Visit the Town Square channel
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Post few messages
+        postMessages(count);
+    });
+
+    it('MM-24051 Verify Keyboard support in Search Results', () => {
```

Comment:
Do each test case in the TM4J? If yes, can we start using test case ID, e.g. `MM-T1492`? It's is new and not yet documented but it will make it easier to integrate between Cypress and TM4J in the future.

Reply:
Sure @saturninoabril. We can start using this form in the future tests. Currently I have already created JIRA tickets for these tests and had kind of combined few tests. But in future, we can start referring the TM4J ID directly.